### ISSUE TITLE
Is there a way to delete a created link?

### ISSUE THREAD

**Issue by AlexMini2517** [NayamAmarshe/MagLit#26]

**Comment by NayamAmarshe** [comment]
Yes there is. You can email me the link at `maglit-admin@protonmail.com`. Deleting links is kinda hard because I have no way to verify whether the link was created by you or not, so I have to decide it on my own.
